src/test/java/org/isf/patient/test/Tests.java

```diff
@@ -232,6 +278,448 @@ public void testMergePatientHistory() throws Exception {
+		testPatient.check(patients.get(patients.size() - 1));
+	}
+
+	@Test
+	public void testMgrGetPatientsByOneOfFieldsLike() throws Exception {
+		_setupTestPatient(false);
+		// Pay attention that query return with PAT_ID descendant
+		ArrayList<Patient> patients = patientBrowserManager.getPatientsByOneOfFieldsLike(null);
+		testPatient.check(patients.get(0));
+	}
+
+	@Test
+	public void testMgrGetPatientsByOneOfFieldsLikeFirstName() throws Exception {
+		Integer code = _setupTestPatient(false);
+		Patient foundPatient = patientIoOperation.getPatient(code);
+		ArrayList<Patient> patients = patientBrowserManager.getPatientsByOneOfFieldsLike(foundPatient.getFirstName());
+		testPatient.check(patients.get(0));
+	}
+
+	@Test
+	public void testMgrGetPatientsByOneOfFieldsLikeMiddleOfFirstName() throws Exception {
+		Integer code = _setupTestPatient(false);
+		Patient foundPatient = patientIoOperation.getPatient(code);
+
+		ArrayList<Patient> patients = patientBrowserManager
+				.getPatientsByOneOfFieldsLike(foundPatient.getFirstName().substring(1, foundPatient.getFirstName().length() - 2));
+		testPatient.check(patients.get(0));
+	}
+
+	@Test
+	public void testMgrGetPatientsByOneOfFieldsLikeSecondName() throws Exception {
+		Integer code = _setupTestPatient(false);
+		Patient foundPatient = patientIoOperation.getPatient(code);
+		ArrayList<Patient> patients = patientBrowserManager.getPatientsByOneOfFieldsLike(foundPatient.getSecondName());
+		testPatient.check(patients.get(0));
+	}
+
+	@Test
+	public void testMgrGetPatientsByOneOfFieldsLikeNote() throws Exception {
+		Integer code = _setupTestPatient(false);
+		Patient foundPatient = patientIoOperation.getPatient(code);
+		ArrayList<Patient> patients = patientBrowserManager.getPatientsByOneOfFieldsLike(foundPatient.getSecondName());
+		testPatient.check(patients.get(0));
+	}
+
+	@Test
+	public void testMgrGetPatientsByOneOfFieldsLikeTaxCode() throws Exception {
+		Integer code = _setupTestPatient(false);
+		Patient foundPatient = patientIoOperation.getPatient(code);
+		ArrayList<Patient> patients = patientBrowserManager.getPatientsByOneOfFieldsLike(foundPatient.getTaxCode());
+		testPatient.check(patients.get(0));
+	}
+
+	@Test
+	public void testMgrGetPatientsByOneOfFieldsLikeNotExistingStringShouldNotFindAnything() throws Exception {
+		_setupTestPatient(false);
+		ArrayList<Patient> patients = patientBrowserManager.getPatientsByOneOfFieldsLike("dupa");
+		assertThat(patients).isEmpty();
+	}
+
+	@Test
+	public void testMgrGetPatientByName() throws Exception {
+		Integer code = _setupTestPatient(false);
+		Patient foundPatient = patientIoOperation.getPatient(code);
+		Patient patient = patientBrowserManager.getPatientByName(foundPatient.getName());
+		assertThat(patient.getName()).isEqualTo(foundPatient.getName());
+	}
+
+	@Test
+	public void testMgrGetPatientByNameDoesNotExist() throws Exception {
+		assertThat(patientBrowserManager.getPatientByName("someUnusualNameThatWillNotBeFound")).isNull();
```

Comment:
same than previous comment , since the method getPatientByName(String) from the type PatientBrowserManager is deprecated, what is the plan ?

Reply:
See above.